rubrik_cdm/cluster.py

```diff
@@ -606,3 +608,56 @@ def read_only_authorization(self, username, timeout=15):
 
         self.log("read_only_authorization: Granting read-only privilages to user '{}'.".format(username))
         return self.post("internal", "/authorization/role/read_only_admin", config, timeout)
+
+    def cluster_node_id(self,timeout=15):
+        """ Returns a list of node ids from all the nodes in the cluster.
+
+                Arguments:
+                    None
+
+                Keyword Arguments:
+                    None
+
+                Returns:
+                    dict -- The full API response from `POST /internal/node`.
+                """
+        self.log("cluster_node_id - Getting all the node ids from the from the cluster")
+        node_id = self.get('internal', '/node',timeout)
+        return node_id
```

Comment:
Just return the API call directly instead of assigning it to a variable first. Also, make sure you have a space between the ",timeout" (i.e ", timeout")

`return self.get('internal', '/node', timeout)`